src/cli.ts

```diff
@@ -72,29 +74,36 @@ import { rmSync } from 'fs'
     }
   })
 
-  program.command('check [message]').action((message) => {
-    if (stdin) {
-      message = stdin
-      if (message) {
-        const config = getConfig()
-        let baselineFile
-        try {
-          baselineFile = readBaselineErrorsFile(config.path)
-        } catch (err) {
-          console.error(
-            `
-Unable to read the .tsc-baseline.json file at "${config.path}".
-
-Has the baseline file been properly saved with the 'save' command?
-`
-          )
-          process.exit(1)
-        }
-        if (!isBaselineVersionCurrent(baselineFile)) {
-          const baselineFileVersion = getBaselineFileVersion(baselineFile)
-          if (baselineFileVersion < CURRENT_BASELINE_VERSION) {
+  program
+    .command('check [message]')
+    .addOption(
+      new Option('--error-format [error-format]', 'Specifies the format for outputting errors.')
```

Comment:
Nice. I like that his option is only available for the `check` command. Makes sense.